test/test_serialization.py

```diff
@@ -673,6 +673,15 @@ def test(name_or_buffer):
 
         test(io.BytesIO())
 
+    # Ensure large zip64 serialization works properly
+    def test_serialization_2gb_file(self):
```

Comment:
I recall having a test for big zip files but it got disabled for being too slow :(